src/azure-cli/azure/cli/command_modules/synapse/operations/sqlpool.py

```diff
@@ -27,3 +28,139 @@ def update_sql_pool(cmd, client, resource_group_name, workspace_name, sql_pool_n
     sku = Sku(name=sku_name)
     sql_pool_patch_info = SqlPoolPatchInfo(sku=sku, tags=tags)
     return client.update(resource_group_name, workspace_name, sql_pool_name, sql_pool_patch_info)
+
+
+def restore_sql_pool(cmd, client, resource_group_name, workspace_name, sql_pool_name, dest_name, performance_level=None,
```

Comment:
>dest_name [](start = 86, length = 9)

`destination_name` may be better to me

Reply:
Thanks I have changed it to `destination_name`  and add `--dest-name` in options list.

---
In reply to: [533103965](https://github.com/Azure/azure-cli/pull/16095#discussion_r533103965) [](ancestors = 533103965)